packages/ses/README.md

```diff
@@ -55,6 +60,10 @@ console.log(Object.isFrozen([].__proto__));
 // true
 ```
 
+Lockdown does not erase any powerful objects from the initial global scope,
+but lays a foundation for controlled access to **capabilities** like access to
+the file system through Compartments.
```

Comment:
Long again. How about "Lockdown does not remove any objects from the initial global scope. It does lay a foundation for controlled **capability** access, such as file system access through Compartments."